src/main/java/io/jenkins/plugins/oidc_provider/Keys.java

```diff
@@ -83,26 +94,33 @@ public JSONObject doDynamic(StaplerRequest req) {
     }
 
     static JSONObject openidConfiguration(String issuer) {
+        JSONArray algValuesSupported = new JSONArray();
+
+        for (SupportedKeyAlgorithm value : SupportedKeyAlgorithm.values()) {
+            algValuesSupported.element(value.name());
+        }
+
         return new JSONObject().
             accumulate("issuer", issuer).
             accumulate("jwks_uri", issuer + JWKS).
             accumulate("response_types_supported", new JSONArray().element("code")).
             accumulate("subject_types_supported", new JSONArray().element("public")).
-            accumulate("id_token_signing_alg_values_supported", new JSONArray().element("RS256")).
+            accumulate("id_token_signing_alg_values_supported", algValuesSupported).
```

Comment:
I am a bit confused by the documentation, since it mentions 
> The algorithm RS256 MUST be included
Does this mean a `RS256` key must be present all the time? 

Due to this, I went to listing all supported types by the plugin, but I can change it :slightly_smiling_face: